src/index.js

```diff
@@ -0,0 +1,160 @@
+const state = {
+    lat: 33.7488,
+    lon: 84.3877,
+    city: "Seattle"
+};
+
+document.addEventListener("DOMContentLoaded", function () {
+    // ------------- wave 2: increase and decrease temp ------------------
+    const increaseTemp = document.querySelector("#increase-temp");
+    const decreaseTemp = document.querySelector("#decrease-temp");
+    const displayTemp = document.querySelector("#display-temp");
+    const resetButton = document.querySelector("#reset-button"); 
+    const searchButton = document.querySelector("#search-button");
+    const cityName = document.getElementById("city-name");
+    const cityInput = document.getElementById("city-input");
+    const selectSky = document.querySelector("#sky-dropdown");
+    const result = document.querySelector("#sky");
+    const landscape = document.querySelector("#landscape");
+
+    
+    // ----- changes temp(converted to fahr) -> landscape -> text color ------
+    let temperature = 65;
+    let lands = "🌾🌾   🍃 🪨    🛤  🌾🌾🌾  🍃";
```

Comment:
These could be things that you could be put in your `state` variable, you want to use your state to hold information specific to a webpage at a specific time.